pyrit/scenarios/atomic_attack.py

```diff
@@ -153,6 +153,15 @@ def __init__(
             f"attack type: {type(attack).__name__}, context type: {self._context_type}"
         )
 
+    def get_objectives(self) -> List[str]:
+        """
+        Get a copy of the objectives list for this atomic attack.
+
+        Returns:
+            List[str]: A copy of the objectives list.
+        """
+        return list(self._objectives)
+
```

Comment:
```suggestion
    @property
    def objectives(self) -> List[str]:
        return list(self._objectives)
```
Nitpick as it's slightly more Pythonic but otherwise identical